Project-Aurora/Project-Aurora/Devices/OpenRGB/OpenRGBDevice.cs

```diff
@@ -1,30 +1,30 @@
-﻿using Aurora.Settings;
+﻿using Aurora.Modules.ProcessMonitor;
+using Aurora.Settings;
+using CSScripting;
+using Microsoft.Scripting.Utils;
 using OpenRGB.NET;
 using System;
 using System.Collections.Generic;
 using System.ComponentModel;
 using System.Linq;
 using System.Threading;
-using Aurora.Modules.ProcessMonitor;
-using Microsoft.Scripting.Utils;
 using Color = System.Drawing.Color;
-using DK = Aurora.Devices.DeviceKeys;
+using OpenRGBClient = OpenRGB.NET.OpenRgbClient;
 using OpenRGBColor = OpenRGB.NET.Color;
-using OpenRGBDevice = OpenRGB.NET.Device;
 using OpenRGBDeviceType = OpenRGB.NET.DeviceType;
 using OpenRGBZoneType = OpenRGB.NET.ZoneType;
 
 namespace Aurora.Devices.OpenRGB
 {
-    public class OpenRgbAuroraDevice : DefaultDevice
+    public class OpenRGBDevice : DefaultDevice
     {
         public override string DeviceName => "OpenRGB";
-        protected override string DeviceInfo => string.Join(", ", _devices.Select(d => d.OrgbDevice.Name));
+        protected override string DeviceInfo => string.Join(", ", _helpers.Select(h => h.Device.Name));
 
-        private OpenRgbClient _openRgb;
-        private List<HelperOpenRgbDevice> _devices;
+        public readonly Queue<DeviceKeys> MouseLights = new Queue<DeviceKeys>(OpenRgbKeyNames.MouseLights);
 
-        private readonly object _updateLock = new();
```

Comment:
is this okay? Race conditions might break some stuff while device list is updating. Can you do "Rescan devices" on OpenRGB while Aurora is running a few times?